letsencrypt/config.json

```diff
@@ -58,7 +58,11 @@
       "aws_access_key_id": "str?",
       "aws_secret_access_key": "str?",
       "sakuracloud_api_token": "str?",
-      "sakuracloud_api_secret": "str?"
+      "sakuracloud_api_secret": "str?",
+      "netcup_customer_id": "str?",
+      "netcup_api_key": "str?",
+      "netcup_api_password": "str?",
+      "netcup_propagation_seconds": "str?"
```

Comment:
why not use constant 300s?

Reply:
I think it's best when users have an option to tune that setting. The netcup certbot authenticator also gives you the option to set it by yourself. Or am I getting you wrong?

Reply:
Please explain me the benefit of this option in comparson to set a fix value of 600s or whatever.

Reply:
According to the developer of the certbot netcup plugin, sometimes 600 is not high enough. Its best, when a user can find a value that works for him. Of course I could just set it to a really high value, since this service is running non-interactive. But question is: why remove this option for a user?

Reply:
okay. Please use the correct type like `int(min=60, max=900)?`